app/actions/remote/thread.ts

```diff
@@ -286,15 +285,10 @@ export const syncThreadsIfNeeded = async (serverUrl: string, isCRTEnabled: boole
             return {models: []};
         }
 
-        const {database, operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
+        const {operator} = DatabaseManager.getServerDatabaseAndOperator(serverUrl);
         const promises = [];
         const models: Model[][] = [];
 
-        // this is to keep backwards compatibility with servers that send the
-        // threads for DM / GM regardless for every team
-        const version = await getConfigValue(database, 'Version');
-        const hasThreadExclusions = isMinimumServerVersion(version, 10, 2, 0);
-
```

Comment:
I can understand changes in the file /actions/remote/entry/common.ts, but what do these changes have anything to do with not re-fetching data for the initial team?

Reply:
The initial team selected first is already fetching its data, here we were not filtering this fetching the same data again